Code/src/main/java/nl/utwente/viskell/ui/components/Block.java

```diff
@@ -153,20 +153,20 @@ public void handleConnectionChanges(boolean finalPhase) {
         }
     }
     
-    /** @return A pair containing the expression this block represents and a set of required blocks. */
-    public abstract Pair<Expression,Set<Block>> getLocalExpr();
+    /** @return A pair containing the expression this block represents and a set of out-of-reach OutputAnchors. */
```

Comment:
Maybe explain in the Javadoc why getLocalExpr needs to know about out-of-reach OutputAnchors.